src/templates/project_duplication_request_completed.html

```diff
@@ -9,7 +9,7 @@
                 Project
                 <a href="https://www.morphobank.org/myprojects/$projectId/overview" target="_blank">$projectId</a>
                 has been duplicated to Project
-                <a href="https://www.morphobank.org/myprojects/$clonedProjectId/overview" target="_blank">clonedProjectId</a>.
+                <a href="https://www.morphobank.org/myprojects/$clonedProjectId/overview" target="_blank">$clonedProjectId</a>.
```

Comment:
For this and above link, please use env variable for host/domain, otherwise it will be confusing when QA